include/log++.h

```diff
@@ -352,11 +352,19 @@ LPP_INTL::InternalPolicyLog(LPP_GET_KEY(), n, LPP_INTL::BaseSeverity::DEBUG, LPP
 #define DLOG_IF_EVERY_N(severity, condition, n) LPP_ASSERT_GLOG(LPP_INTL::GlogSeverity::severity); if (condition) LPP_INTL::InternalPolicyLog(LPP_GET_KEY(), n, LPP_INTL::BaseSeverity::DEBUG, LPP_INTL::PolicyType::EVERY_N)
 #define LOG_STRING(severity, ptr) LPP_ASSERT_GLOG(LPP_INTL::GlogSeverity::severity); LPP_INTL::InternalGlogLogStringLog(toBase(LPP_INTL::GlogSeverity::severity), ptr)
 
-#ifndef GLOG_SUPPORTED
-inline static int32_t FLAGS_v;
-#define VLOG_IS_ON(verboselevel) FLAGS_v >= (verboselevel) ? true : false
+/**
+ * Replace glog's FLAGS_v and VLOG_IS_ON to avoid linker errors
+ * if glog is installed but not linked to lpp.
+ */
+inline static int32_t LPP_FLAGS_v;
```

Comment:
I'm getting the warning
```
/home/rik/pq_ws/src/lpp/include/log++.h:359:23: warning: ‘LPP_FLAGS_v’ defined but not used [-Wunused-variable]
  359 | inline static int32_t LPP_FLAGS_v;
```

Do you think we can do something about it?

Reply:
Yes, I added the [[maybe_unused]] C++ attribute